### PR TITLE
Fix modal height in firefox

### PR DESCRIPTION
**Pull request by ahanipeitra1994**
Hi this is fix of the issue #1590. I added a height of 100% important.

### CODE REVIEW

File: assets/css/iziModalToast/iziModal.css [hunk 1]
```diff
@@ -244,6 +244,7 @@
     margin: 0 0 -6px 0;
     width: 100%;
     transition: height 0.3s ease;
+    height: 100% !important;
```
Comment (muarachmann) [inline assets/css/iziModalToast/iziModal.css]: Lets give it `95%`

**Comment by Ngai-E** [comment]
ok get the fix in

File: assets/css/iziModalToast/iziModal.css (hunk 1 above)
Comment (ahanipeitra1994) [reply assets/css/iziModalToast/iziModal.css]: Ok i will do that and see how it looks like thanks @muarachmann

**Comment by ahanipeitra1994** [comment]
Muarachman I have changed the height to 95% .

**Comment by muarachmann** [comment]
> Muarachman I have changed the height to 95% .

Thanks

**Pull request merged**